### PR TITLE
Define and activate AWS 12.1.4 release

### PR DESCRIPTION
**Pull request by sslavic**
Towards https://github.com/giantswarm/giantswarm/issues/12992

### CODE REVIEW

File: aws/v12.1.4/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# :zap: Giant Swarm Release v12.1.4 for AWS :zap:
+
+This release upgrades external-dns app to v1.4.0.
```
Comment (cokiengchiara) [inline aws/v12.1.4/README.md:3]:
```suggestion
This release upgrades external-dns app to v1.4.0 to improve observability.
```

**Comment by cokiengchiara** [comment]
@alex-dabija just asking for review since it's AWS :) I checked and all the notes to SEs, etc. all still seem relevant.

**Comment by sslavic** [comment]
Tested on giraffe
- 12.1.4 cluster creation
- upgrade from 12.1.3
- upgrade from 11.5.3

**Review by alex-dabija — APPROVED** [review]

**Comment by sslavic** [comment]
@cokiengchiara we have ack from Alex, release has been tested. Are you OK with release notes? Does announcement need to be tested? I.e. can we ship? Same Q for the 11.5.4 release.

**Comment by sslavic** [comment]
Seems @cokiengchiara is busy but OK to ship https://gigantic.slack.com/archives/C02521EC0/p1598613561024800?thread_ts=1598613462.024600&cid=C02521EC0

Merging.

**Pull request merged**